lib/Qandidate/Toggle/Serializer/OperatorSerializer.php

```diff
@@ -69,7 +70,7 @@ public function deserialize(array $operator)
             case 'equals-to':
```

Comment:
Given we "fix" `equals-to` above. Let's add the `equal-to` case here, but also keep the old one for backwards compatibility.